python-pytest.el

```diff
@@ -154,6 +154,9 @@ When non-nil only ‘test_foo()’ will match, and nothing else."
     ("d" "Test def/class (dwim)" python-pytest-function-dwim)
     ("D" "Test this def/class" python-pytest-function)]])
 
+(define-obsolete-function-alias 'python-pytest-popup
```

Comment:
nice, this will help for people who have keyboard shortcuts to the old function, and i know for sure i am not the only person who does that. :+1:

Reply:
I have this as well. And it's actually working nicely. `helpful` is reporting the function name pretty nicely.